NEXT_CHANGELOG.md

```diff
@@ -14,5 +14,6 @@ Previously ".internal" folder under artifact_path was not cleaned up as expected
 
 ### Bundles
 * Fixed cleaning up artifact path .internal folder ([#2572](https://github.com/databricks/cli/pull/2572))
+* Do not use app config section in test templates and generated app configuration ([#2599](https://github.com/databricks/cli/pull/2599))
```

Comment:
More understandable for customers. We don't need to call out the test template changes.

```suggestion
* Use app.yml instead of inlining config when using `bundle generate` command for apps ([#2599](https://github.com/databricks/cli/pull/2599))
```